packages/terra-clinical-result/src/result-time-header-cell/ResultTimeHeaderCell.jsx

```diff
@@ -61,8 +62,9 @@ const ResultTimeHeaderCell = (props) => {
       {...customProps}
       className={timeHeaderCellClassnames}
     >
-      <div className={dateClassnames}>{date}</div>
-      <div className={cx('time')}>{time}</div>
+      <time className={dateClassnames}>{date}</time>
+      <VisuallyHiddenText className={cx('hidden-space')} text={'\u00A0'} />
```

Comment:
Is the className attribute needed for this component if it's hidden?

Reply:
I was using the className [here](https://github.com/cerner/terra-clinical/pull/872/files#diff-e41002d62a16236c97fa2e987e65bbefee808c6bf73b9dcffec4abdf25ce3fc5R43-R44) to maintain this selector that is setting the `margin-top` value on all elements with the `.time` class but I think we can just move this style into that class.

Reply:
Removed here: https://github.com/cerner/terra-clinical/pull/872/commits/06e0278063bf39c26cf889b6aaad6714db468455